config/spryk/spryks/Spryker/Zed/Persistence/Propel/Schema/AddZedPersistencePropelSchemaProperty.yml

```diff
@@ -11,18 +10,25 @@ arguments:
         inherit: true
         default: Spryker
 
-    name:
-        inherit: true
-
     propertyName: ~
 
     propertyType: ~
 
-    singular:
-        default: false
+    required: ~
```

Comment:
Add useful defaults to those arguments. You can see what happens if we don't have good default values in the AddCrudFacade spryk, many arguments need to be set there to be just empty.